CONTRIBUTING.md

```diff
@@ -0,0 +1,153 @@
+# Development Ideology
+
+Truths which we believe to be self-evident (adapted from [TextSecure's](https://github.com/WhisperSystems/TextSecure/blob/master/contributing.md))
+
+1. **The answer is not more options.** If you feel compelled to add a preference that's exposed to the user, it's very possible you've made a wrong turn somewhere.
+2. **There are no power users.** The idea that some users "understand" concepts better than others has proven to be, for the most part, false. If anything, "power users" are more dangerous than the rest, and we should avoid exposing dangerous functionality to them.
+3. **If it's "like PGP," it's wrong.** PGP is our guide for what not to do.
```

Comment:
1. We get into a *lot* of trouble with all the myriad of interacting options that we expose to the user. By all means it makes sense to have plugable strategies which are extension points that users can inject into their instance by installing the plugin. But when you expose these options to *every* user then that is a usability failure. I want people to stop and think before they add just another checkbox.
2. If we need to add options "for power users" then we are doing something wrong again... we should be designing the UI so that features are easily accessible to all. The proliferation of `RunScripts` hacks point to a deeper issue.
3. If you are adding a CLI command and you make it's options like the PGP command lines... ![](http://cdn.meme.am/instances2/500x/1260668.jpg) (Not literally, but figuratively)